backend/.golangci.yml

```diff
@@ -0,0 +1,22 @@
+run:
+  tests: false
+  output:
+    format: tab
+  skip-dirs:
+    - vendor
+linters:
+  enable:
+    - unconvert
+    - megacheck
+    - structcheck
+    - gas
+    - gocyclo
+    - dupl
+    - misspell
+    - unparam
+    - varcheck
+    - deadcode
+    - typecheck
+    - ineffassign
+    - varcheck
+  disable-all: false
```

Comment:
I believe it's `true` in original. Other than that, LGTM.

Reply:
changed